WordPress/src/main/java/org/wordpress/android/sharedlogin/WordPressPublicData.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.sharedlogin
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.sharedlogin.WordPressPublicData.PackageName.Jalapeno
+import org.wordpress.android.sharedlogin.WordPressPublicData.PackageName.Vanilla
+import org.wordpress.android.sharedlogin.WordPressPublicData.PackageName.Wasabi
+import javax.inject.Inject
+
+class WordPressPublicData @Inject constructor() {
+    private sealed class PackageName(val type: String, val value: String) {
+        object Jalapeno : PackageName("jalapeno", "org.wordpress.android.prealpha")
+
+        object Vanilla : PackageName("vanilla", "org.wordpress.android")
+
+        object Wasabi : PackageName("wasabi", "org.wordpress.android.beta")
+    }
+
+    fun currentPackageId(): String = when (BuildConfig.FLAVOR_buildType) {
+        Jalapeno.type -> Jalapeno.value
+        Vanilla.type -> Vanilla.value
+        Wasabi.type -> Wasabi.value
+        else -> throw IllegalArgumentException("Failed to get Jetpack package ID: build flavor not found.")
+    }
```

Comment:
That only works because WP and JP currently share a single `build.gradle` file and thus have the same list of buildType flavors declared in that `build.gradle` for both WP and JP, right? And would break if one day in the future we start splitting the WP vs JP code a bit more and make Jetpack a separate project (root folder next to WordPress) with its own `build.gradle` possibly using different build flavors…

Again, probably not a problem for today so this should be ok for now, but kinda made me think of how this couples the buildType flavors of both products in a way…